addon/components/au-accordion.gts

```diff
@@ -107,15 +110,15 @@ export default class AuAccordion extends Component<AuAccordionSignature> {
               @alignment="left"
               @size="large"
               @ariaHidden={{true}}
-              data-test-accordion-icon-open={{this.iconOpen}}
+              data-test-accordion-icon-open={{iconTestName this.iconOpen}}
```

Comment:
Hmmm, I think we should just remove the attribute and corresponding test to be honest. Adding an extra helper call simply to test something seems unneeded, and I'm not convinced it was a good test in the first place (it probably makes more sense to use visual regression tests for something like this).

Also, was this helper wrapper needed? I would have expected that Ember would toString the value since it's an attribute. Or was this for type reasons maybe?

Reply:
I can also do that in master already, after which you can rebase? It seems unrelated to this PR, I just noticed it here. Thoughts?

Reply:
Yeah, makes sense to me to remove it separately and rebase on that.

Reply:
...and done.